acoustid.py

```diff
@@ -179,10 +180,14 @@ def _api_request(url, params, timeout=None):
         "Content-Type": "application/x-www-form-urlencoded"
     }
 
+    params['meta'] = '+'.join(params['meta'])
+
     with requests.Session() as session:
         session.mount('http://', CompressedHTTPAdapter())
         try:
-            response = session.post(url, data=params, headers=headers,
+            response = session.post(url, 
+                                    data=urlencode(params, safe='+'),
```

Comment:
And when you do the change above, you can revert this one.

Reply:
I've did some testing and it turns out: Of course, it doesn't matte if the `+` is safed or whitespace is encoded. But another thing showed up: When using the list it is irrelevant if `+` or whitespace is used to concat the items. But if the user brings an already concated string using the `+` as it is shown in the AcoustID Web Service documentation the original problem would persist if I revome this explicit `urlencode` to safe the `+`. The `+` is still encoded as `%20B` which leads the AcoustID Web Service to ignore the `meta` parameter.

Reply:
Thanks for explaining! TBH I think it's OK to just make the only "supported" way of passing multiple values here a list… that way we sidestep the issue. I'll simplify this a bit and then merge!